x-pack/plugins/endpoint/common/types.ts

```diff
@@ -23,20 +23,56 @@ export type ImmutableSet<T> = ReadonlySet<Immutable<T>>;
 export type ImmutableObject<T> = { readonly [K in keyof T]: Immutable<T[K]> };
 
 export class EndpointAppConstants {
+  static ALERT_INDEX_NAME = 'my-index';
   static ENDPOINT_INDEX_NAME = 'endpoint-agent*';
 }
 
+export interface AlertResultList {
+  /* the alerts restricted by the page size */
```

Comment:
Could we change this comments to JSDoc style comments? Like this:
```
/**
 * the alerts restricted by the page size
 */
```

Reply:
Sure!